src/OSPSuite.Core/Chart/ChartWithObservedData.cs

```diff
@@ -37,9 +39,12 @@ public void RemoveObservedData(DataRepository dataRepository)
          RemoveCurvesForDataRepository(dataRepository);
       }
 
-      public virtual void UpdateFrom(ChartWithObservedData originalChart)
+      public override void UpdatePropertiesFrom(IUpdatable source, ICloneManager cloneManager)
```

Comment:
@Yuri05 Adding this here to support some basic update when cloning chart. It was not derived from ObjectBase..not sure why